dbc/dbc.py

```diff
@@ -0,0 +1,56 @@
+# Python Script to convert all IDs in a DBC file to a can_recv_entries.h file
+
+# run pip install cantools
+
+# test --> python3 dbc.py examples/tessie.dbc 0x01 10 test_write
+
+import cantools
+import sys
+import os
+from pathlib import Path
+
+bold = "\033[1m"
+end = "\033[0m"
+
+if len(sys.argv) < 5: 
+    print(f"{bold} not enough args (dbc file path, can bus number, default size, output dir) {end}")
+    exit();
+
+DBC_FILE_PATH = Path(sys.argv[1])
+CAN_BUS = Path(sys.argv[2])
+DEFAULT_SIZE = Path(sys.argv[3])
+WRITE_FILE_DIR = Path(sys.argv[4])
```

Comment:
output dir can be an optional argument with default being current directory. look at argparse library to make this easier